netket/tools/check_mpi.py

```diff
@@ -33,6 +35,7 @@ def check_mpi():
     info = {
         "mpi_available": mpi_available,
         "mpi4jax_available": mpi4jax_available,
+        "avalable_cpus (rank 0)": available_cpus(),
```

Comment:
technically this should be called `subscriptable_cpus (rank 0)` but I think that's too much jargonic for users?

Reply:
Yeah, I agree - available is easier to understand.